### PR TITLE
More Admin Layout Improvements - For mobile screens

### PR DESCRIPTION
**Pull request by daltonfury42**
Social Media icons were not wrapping properly, they were going off screen:
image

### CODE REVIEW

File: simplq/src/components/pages/Admin/Token.jsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ function Token(props) {
       </div>
       <div className={styles['token-details']}>
         <div className={styles['token-name-time']}>
-          <p>Join time 11:02 pm</p>
```
Comment (daltonfury42) [inline simplq/src/components/pages/Admin/Token.jsx:108]: Removing joining time as it's taking up more space... but this I was not able to completely solve still... it's not very straight forward:

image


We need to flex wrap properly or do something on small screens...

**Comment by maaverik** [comment]
It looks like this messes up the centering on mobile view which is present on the main site.

Screenshot from 2020-11-01 16-28-08

**Comment by daltonfury42** [comment]
got it.. can I fix this as a follow up PR?

**Comment by maaverik** [comment]
Okay

**Review by maaverik — APPROVED** [review]

**Pull request merged**